public/online-player.js

```diff
@@ -4,11 +4,11 @@ var CANVAS_HEIGHT = 2000;
 function OnlinePlayer(id, name, x, y, color) {
```

Comment:
Take color out, we don't pass this in. It should default to 'white' and we update with a new color when the user submits that info.